src/util/loaders.ts

```diff
@@ -148,3 +61,26 @@ export const loadEvents = async (client: Client, events: DiscordEvent[]): Promis
     });
   }
 };
+
+/**
+ *
+ * @returns An array of events
+ */
+const loadEvents = (): DiscordEvent[] => {
+  const events = [hasVarEvent, readyEvent, justAskEvent, interactionCreateEvent].filter(
+    eventPredicate
+  );
+  return events as DiscordEvent[];
+};
+
+/**
+ *
+ * @returns A map of command names to commands
+ */
+const loadCommands = (): Map<string, Command> => {
+  const commands = [pingCommand, tipsCommands, guidesCommand, docsCommands].flat();
+  return new Map(commands.filter(commandPredicate).map((command) => [command.data.name, command]));
+};
+
+export const commands = loadCommands();
+export const events = loadEvents();
```

Comment:
(suggestion) as mentioned in my other comment, use a barrel file. in the barrel file you prepare the data as you need it.
so the loadCommands function then becomes this in the barrel file
```ts
export const commands = new Map([pingCommand, tipsCommands, guidesCommand, docsCommands].flat().map((command) => [command.data.name, command]));
```